config/main.py

```diff
@@ -42,13 +45,13 @@ def run_command(command, display_cmd=False, ignore_error=False):
 def interface_alias_to_name(interface_alias):
     """Return default interface name if alias name is given as argument
     """
-
-    cmd = 'sonic-cfggen -d --var-json "PORT"'
-    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
-
-    port_dict = json.loads(p.stdout.read())
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    port_dict = config_db.get_table('PORT')
 
     if interface_alias is not None:
+        if not port_dict:
+            click.echo("port_dict is None!")
```

Comment:
If port_dict is None, then the next line will crash. Should you do something else here in addition to printing a message in order to handle this gracefully?